Samples/AudioPreprocessing/WinUI/AudioPreprocessing/AudioPreprocessing/Model/PreprocessingModel.cs

```diff
@@ -0,0 +1,244 @@
+﻿using Microsoft.AI.MachineLearning;
+using Microsoft.AI.MachineLearning.Experimental;
+using NAudio.Wave;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Windows.Graphics.Imaging;
+using Windows.Media;
+using Operator = Microsoft.AI.MachineLearning.Experimental.LearningModelOperator;
+
+namespace AudioPreprocessing.Model
+{
+    public class PreprocessModel
+    {
+        const string MicrosoftExperimentalDomain = "com.microsoft.experimental";
+
+        public string AudioPath { get; set; }
+
+        public string MelSpecImagePath { get; set; }
+
+        public PreprocessModel()
+        {
+            AudioPath = "..\\tmp\\mel_spectrogram_file.jpg";
+            MelSpecImagePath = "..\\tmp\\mel_spectrogram_file.jpg";
+        }
+
+        public SoftwareBitmap GenerateMelSpectrogram(string audioPath)
+        {
+            var signal = GetSignalFromFile(audioPath);
+            int batchSize = 1;
```

Comment:
love naming your constants instead of hardcoding them in the method call below. 👍 
should they be prefixed with const?

Reply:
Oh hm! Ideally they'll be changeable, via toggles etc. from the UI. It might make more sense to make these default values in the method call.